common/s2n-codec/src/encoder/estimator.rs

```diff
@@ -4,6 +4,9 @@
 use crate::encoder::Encoder;
 
 /// Estimates the `encoding_size` of an `EncoderValue`
+#[cfg_attr(test, derive(Debug))]
+#[cfg_attr(test, derive(Clone))]
+#[cfg_attr(test, derive(bolero::TypeGenerator))]
 pub struct EncoderLenEstimator {
```

Comment:
you only need one `cfg_attr`

```suggestion
#[cfg_attr(test, derive(Clone, Debug, bolero::TypeGenerator))]
pub struct EncoderLenEstimator {
```